doc/SYNDICATION.md

```diff
@@ -0,0 +1,147 @@
+- **Quick Overview**
+
+    Syndication (also known as Republishing), is the term for allowing other publishers to use our content in their own publications. At the time of writing this there are over 200 Syndication licences and nearly 600 end users for those licences. In total those ~200 licences amount to over £3.2m.
+
+- **What do users see**
+    - **Syndication Icons**
+
+        Once the user has syndication licence, [ft.com](http://ft.com) appears sightly different. 
+
+        When on [ft.com](http://ft.com), users see coloured symbols next to the title of every  article/podcast/video on the page.
+        
+        ![ft_homepage.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/ft_homepage.png)
+
+        The symbols indicates different syndication grants:
+
+        ![syndication_symbols.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_symbols.png)
+
+        On clicking on the `green tick` icon, the user can save or download the content
+
+        ![syndication_modal.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/syndication_modal.png)
+
+    - **Republishing Platform**
+
+        On the top right corner on the page, the `Republishing` link takes users to the republishing platform.
+
+        ![republishing_menu.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_menu.png)
+
+        On the republishing tool, Users can see their contract details, allowance details and select their preferred download format.
+
+        ![republishing_platform.png](https://github.com/Financial-Times/next-syndication-api/blob/feat/add-syndication-md/doc/images/republishing_platform.png)
+
+        Users can also manage saved, downloaded and spanish content.
+
+- **Key systems**
+
+    - [**n-syndication**](https://github.com/Financial-Times/n-syndication)
+
+        This is the client side library responsible for adding syndication symbols next to articles/podcast and videos headlines on ft.com. Allowing syndication customers to download or save content for republishing.
```

Comment:
Semantic nitpicking `articles/podcast, and videos headlines` mixes plural and singular, maybe stick to one or the other?